book edits (#3955)
status-im/nim-beacon-chain: docs/the_nimbus_book/src/migration.md

# Migrate from another client

This guide will take you through the basics of how to migrate to Nimbus from another client. *See [here](./migration-options.md) for advanced options*.

**Please take your time to get this right.** Don't hesitate to reach out to us in the `#helpdesk` channel of [our discord](https://discord.gg/j3nYBUeEad) if you come across a stumbling block. We are more than happy to help guide you through the migration process. Given what's at stake, there is no such thing as a stupid question.

!!! info ""

>     Unlike other clients, Nimbus does not require a separate validator client - instead, validators are by default run inside the beacon node process.

Reviewer: ```suggestion
    Unlike other clients, Nimbus does not require a separate validator client - instead, the validators duties can be handled by the beacon node process.
```

Reviewer: The previous wording seemed a bit unusual to me.

Author: this wording lacks a `'` possessive - but I think we say that validators "run" elsewhere as well - it's a bit ambiguous whether "validator" refers to a single key, the role of being a validator in general with one or more keys or something else. Checking the other clients, they also talk about running validators interchangeable with "performing validator duties" and other forms - no clear consensus that I can find - anyway, you're right that it  read wrong, fixed it another way

>     Unlike other clients, Nimbus does not require a separate validator client - instead, validators are by default run inside the beacon node process.

Author: ```suggestion
    Unlike other clients, Nimbus does not require a separate validator client - instead, validators run in the beacon node process.
```
